src/common/details/ai_generative_sdk/ai_client_wrapper.cs

```diff
@@ -19,56 +19,148 @@
 
 namespace Azure.AI.Details.Common.CLI
 {
-    public class PythonSDKWrapper
+    public class AIClientWrapper
     {
+        private static bool s_usePython = true;
```

Comment:
what kinds of things are required still to flip to s_usePython = false, and ultimately remove that code?